modules/UI/videolayout/LocalVideo.js

```diff
@@ -33,6 +33,38 @@ export default class LocalVideo extends SmallVideo {
         this.streamEndedCallback = streamEndedCallback;
         this.container = this.createContainer();
         this.$container = $(this.container);
+        const layout = getCurrentLayout(APP.store.getState());
+
+        switch (layout) {
+        case LAYOUTS.VERTICAL_FILMSTRIP_VIEW:
+            this.$container.css('padding-top', `${1 / interfaceConfig.LOCAL_THUMBNAIL_RATIO * 100}%`);
+            break;
+        case LAYOUTS.HORIZONTAL_FILMSTRIP_VIEW: {
+            const state = APP.store.getState();
+            const { height, width } = state['features/filmstrip'].horizontalViewDimensions.local;
```

Comment:
Hmm. I actually can't reproduce it. Probably there's some race condition. I'm looking at it.

Reply:
My env has it right now and I can repro. Maybe it's the specific steps I'm taking? https://lenny.jitsi.net/ANY_MEETING_NAME_HERE#interfaceConfig.VERTICAL_FILMSTRIP=false

Reply:
Awesome! Thanks! That helped a lot.

The difference was that you're passing the option trough the URL. I was editing interface_config directly.

Reply:
That shouldn't make a difference right? Let me know why that makes a difference. interfaceConfig should be overwritten early to avoid this problem.

Reply:
Sure! This shouldn't make a difference but it is the only difference I see :) i guess what happens is that because of the override the order of the calls may be a little different and triggers the race condition. Will investigate further tomorrow if you think it is necessary.

Reply:
Yes it's necessary. Overrides are how the iframe integration works, right?

Reply:
I will check it out tomorrow! BTW you know that this is already fixed in the latest version, right?

Reply:
We just don't know what why the race condition was happening.

Reply:
I'm glad you made me take another look into this. It turned out that after the fix we don't get the JS error anymore but also the local thumbnail size is 0/0 :) I'm now pushing another fix for this.

Here's what it turn out to be the problem:
For the initial size of the thumbnails we rely on the subscriber that checks `shouldDisplayTileView` value. So in the case you described initially  `shouldDisplayTileView` value changes from `undefined` to `false` because the layout is defined as vertical filmstrip in interface_config. Then the overrides are executed but the value of `shouldDisplayTileView` stays `false` and nothing is triggered from there.

Fix:
I changed the subscriber to check the value of `getCurrentLayout`. Currently this is a little weird because interfaceConfig(the only thing that changes in the use case where we experience the issue)  is not in redux. But it is not a problem because during the overrides we are dispatching actions (at least to set the config in redux) so it should be fine for now. And also if one day we put interfaceConfig in the store this will be the preferred solution.